modules/tls/src/main/java/org/mule/runtime/module/tls/internal/MuleTlsContextFactoryBuilder.java

```diff
@@ -32,6 +35,11 @@
     @Override
     public void initialise() throws InitialisationException
     {
+        if (!initialised.compareAndSet(false, true))
```

Comment:
same reason why TlsContextFactory does the same thing. The initialise needs to be idempotent. Otherwise the bug referenced in the connection provider (but not exclusively that one) could mess this up